contracts/governance/TimelockController.sol

```diff
@@ -51,6 +51,11 @@ contract TimelockController is AccessControl, IERC721Receiver, IERC1155Receiver
      */
     event CallExecuted(bytes32 indexed id, uint256 indexed index, address target, uint256 value, bytes data);
 
+    /**
+     * @dev Emitted when new proposal is scheduled.
+     */
+    event ProposalSalt(bytes32 indexed id, bytes32 salt);
```

Comment:
The issue suggested `ProposalSalt` but the rest of the events are `CallExecuted`, `CallScheduled`, etc.

This should probably be called `CallSalt`. @Amxx?

Reply:
Yes, `CallSalt` would make more sens.